src/bom/bitcode.rs

```diff
@@ -459,7 +460,43 @@ fn attach_bitcode(chan : &mut mpsc::Sender<Option<Event>>,
 
     match process::Command::new("objcopy").args(&objcopy_args).stdout(process::Stdio::piped()).stderr(process::Stdio::piped()).current_dir(cwd).spawn() {
         Err(msg) => {
-            return Err(anyhow::Error::new(BitcodeError::ErrorAttachingBitcode(cwd.to_path_buf(), orig_target.clone(), bc_target.clone(), msg)));
+            match process::Command::new("objcopy")
```

Comment:
Is it practical to factor this out into a separate function that is named something like `check_objcopy_version`? I was already getting nervous about the size and complexity of this code

Reply:
Sure.